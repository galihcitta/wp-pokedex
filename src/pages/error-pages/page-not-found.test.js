import React from "react";
import PageNotFound from "./page-not-found";

let wrapper;

beforeEach(() => {
    wrapper = shallowWithTheme(<PageNotFound />);
});

describe("PageNotFound Component", () => {
    it("renders and create snapshot <PageNotFound />", () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it("renders success <PageNotFound />", () => {
        expect(wrapper.exists()).toBe(true);
    });
});
