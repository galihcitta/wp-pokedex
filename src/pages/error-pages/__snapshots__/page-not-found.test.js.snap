// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Component renders and create snapshot <PageNotFound /> 1`] = `
<styled.div
  noPadding={false}
>
  <styled.h3
    align="center"
    block={true}
  >
    404
  </styled.h3>
</styled.div>
`;
